src/notebooksyncagent.h

```diff
@@ -25,8 +25,8 @@
 
 #include "reader.h"
 
-#include <extendedcalendar.h>
-#include <extendedstorage.h>
+#include <notebook.h>
+#include <sqlitecalendarstorage.h>
```

Comment:
Suppose this is no longer needed?

Reply:
Ah sure, changed for `#include <calendarstorage.h>`. Thanks.